hscontrol/types/node.go

```diff
@@ -192,31 +191,21 @@ func (node *Node) IPsAsString() []string {
 	return ret
 }
 
-func (node *Node) HostInfoAsProtoStruct() *structpb.Struct {
-	if node.HostinfoDatabaseField == "" {
-		return nil
+func (node *Node) HostInfoAsString() string {
```

Comment:
maybe this could just be `[]byte` ? not sure if gRPC/protobuf covers that?

Reply:
> maybe this could just be `[]byte` ? not sure if gRPC/protobuf covers that?

[]byte can be returned, but the output will be a base64 string. This will lead to the fact that on the client side you will first have to convert base64 to string, and only then parse it